src/chrome/content/rules/DLsite.com.xml

```diff
@@ -1,44 +1,74 @@
 <!--
+	Related rules:
+	- dlsite.blogimg.jp
```

Comment:
.xml

Reply:
`dlsite.blogimg.jp.xml` or `blogimg.jp.xml` or change that filename to `dlsite.blogimg.jp.xml`?